src/ocean/io/select/EpollSelectDispatcher.d

```diff
@@ -791,9 +800,9 @@ public class EpollSelectDispatcher : IEpollSelectDispatcherInfo
 
                 version ( EpollCounters ) if ( n == 0 ) this.counters.timeouts++;
 
-                auto selected_set = events[0 .. n];
+                this.selected_set = events[0 .. n];
 
-                this.handle(selected_set, this.unhandled_exception_hook);
+                this.handle(this.selected_set, this.unhandled_exception_hook);
```

Comment:
Maybe set `this.selected_set = null` here?

Reply:
Yes, thank you!